src/components/EventList.spec.tsx

```diff
@@ -19,14 +19,21 @@ describe('EventList', () => {
                 dummyEvents = data.data;
             });
         cy.fixture('dummyEventResponses.json')
-            .then((data : {data: ZetkinEvent[]}) => {
-                dummyEvents = data.data;
+            .then((data : {data: ZetkinEventResponse[]}) => {
+                dummyEventResponses = data.data;
             });
     });
 
     it('contains data for each event', () => {
+        const spyOnSubmit = cy.spy();
```

Comment:
The naming is off, implies that there is an `onSubmit`-callback. Is it a rest product from another test?

Reply:
Also, there should be no need for a spy in this test (or the subsequent ones) where the spy is not actually used *for spying*.